include/libopencm3/pac55xx/ccs.h

```diff
@@ -133,6 +218,151 @@ typedef enum {
 	CCS_DSR_DS_25MA = 0x07,
 } ccs_drive_strength_t;
 /**@}*/
+
+/**@}*/
+
+BEGIN_DECLS
+
+/**
+ * @defgroup ccs_api Clock Control System API
+ * @ingroup peripheral_apis
+ * @brief <b>PAC5xx CCS Driver</b>
+ * @author @htmlonly &copy; @endhtmlonly 2020 Kevin Stefanik <kevin@allocor.tech>
+ * @date March 7, 2020
+ *
+ * This library supports the CCS module in the PAC55xx SoC from Qorvo.
+ *
+ * LGPL License Terms @ref lgpl_license
+ */
+
+/**@{*/
+
+/**
+ * Select the source for FRCLK.
+ * @param[in] sel  One of ccs_ctl_frclkmuxsel_t.
```

Comment:
if that's a type, why isn't it the argument type? if it's not a type, why does it end in _t?  (does the docs autolink that or do you need a @ref?)

Reply:
looks like a remnant that needs removal.  I had a few things as a `typedef enum` that were changed to standard macros instead.